src/main/java/com/uid2/shared/secure/AzureCCAksCoreAttestationService.java

```diff
@@ -0,0 +1,31 @@
+package com.uid2.shared.secure;
+
+import com.uid2.shared.Utils;
+import com.uid2.shared.secure.azurecc.IMaaTokenSignatureValidator;
+import com.uid2.shared.secure.azurecc.IPolicyValidator;
+import com.uid2.shared.secure.azurecc.MaaTokenSignatureValidator;
+import com.uid2.shared.secure.azurecc.PolicyValidator;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import lombok.extern.slf4j.Slf4j;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+// CC stands for Confidential Container
+@Slf4j
+public class AzureCCAksCoreAttestationService extends AzureCCCoreAttestationServiceBase {
```

Comment:
Do we actually need child classes for these? Could they just be one class that takes in the AZURE_CC_PROTOCOL?